server/src/test/java/io/unitycatalog/server/base/BaseServerTest.java

```diff
@@ -81,6 +125,21 @@ public void tearDown() {
       session.close();
 
       unityCatalogServer.stop();
+      if (testDirectoryRoot != null && Files.isDirectory(testDirectoryRoot)) {
```

Comment:
Maybe we can use the `@TempDir` annoation in this class, then we don't need to worry about root path deletion.